Tests/Kernels/GraphicTest/Kernel.cs

```diff
@@ -3,7 +3,6 @@
 using Cosmos.TestRunner;
```

Comment:
Can you please also add a few tests for ImageCanvas especially? Maybe call the standard draw tests on a few ImageCanvases.